content/docs/use-cases/index.md

```diff
@@ -1,28 +1,28 @@
 # Use Cases
 
-We provide short articles on common ML workflow or data management scenarios
-that DVC can help with or improve. Our use cases are not written to be run
-end-to-end like tutorials. For more general, hands-on experience with DVC,
-please see our [Get Started](/doc/tutorials/get-started) instead.
+We provide short articles on common ML workflow and data science use cases that
+DVC can help with or improve. Our use cases are not written to be run end-to-end
+like tutorials. For more general, hands-on experience with DVC, please see our
+[Get Started](/doc/tutorials/get-started) instead.
 
 ## Why DVC?
 
 Even with all the success we've seen today in machine learning (ML), especially
-with deep learning and its applications in business, the data science community
-still lacks good practices for organizing their projects and collaborating
-effectively. This is a critical challenge: while ML algorithms and methods are
-no longer tribal knowledge, they are still difficult to implement, reuse, and
-manage.
+with deep learning and its applications in business, data scientists still lack
+good tools for organizing their projects and collaborating effectively. This is
+a critical challenge: while ML algorithms and methods are no longer tribal
```

Comment:
Same Q. Is this SEO related? I'm trying to make sure the PR is well scoped. Looks like copy edits to me, and TBH I'm not very convinced:

- the data science community -> data scientists ?
- practices -> tools ?

Reply:
Yes, also SEO related: trying to associate "data science/scientists" with the terms "tools" and "project organization". I adjusted the original sentence to be more direct by referring to data scientists (specific role instead of general community) and tools (DVC as a tool as opposed to general practice.)

Reply:
I see. Agree about the first change then ("data science community" doesn't seem like a query that gives us any traffic rn anyway) but still not sure about removing "good practices". Well (after checking in the SC), in fact we should change it to "best practices", but that term has potential of it's own I think.

Also speaking of tools here changes the semantics quite a bit, as we're trying to explain part of the motivation for DVC, and that has to do with DS best practices.

So can we change or add text somewhere else to associate "data scientists" and "tools"?

Reply:
Nice! This is even better. You are right about "best practices" — it's definitely another term we should be targeting. Added "tools" in the final bullet. That's a lightweight change and I think it's a better usage, keeping with how DVC is described elsewhere in the docs. Do you agree?

Reply:
Agreed 🤝